lib/mimic.ex

```diff
@@ -418,6 +418,13 @@ defmodule Mimic do
     :ok
   end
 
+  @doc "Returns the current mode (`:global` or `:private`)"
+  @spec mode() :: atom()
```

Comment:
Can you please change the typespec to `@spec mode() :: :global | :private` if that's the only values that can be returned from this function.